manifests/t/Toit/Jaguar/0.8.1/Toit.Jaguar.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Toit.Jaguar
+PackageVersion: 0.8.1
+PackageLocale: en-US
+Publisher: Toitware ApS

```

Comment:
```suggestion
Publisher: (c) 2021 Toitware ApS
```
![image](https://user-images.githubusercontent.com/71691514/153353953-0aea9e87-8384-4be7-a546-d825fb2cbbe7.png)
Also PackageVersion (ie - `AppVersion=` in inno setup) is missing from the installer so please add that to the installer, without it winget cannot determine to whether or not to upgrade the package. (eg - it creates issue like this https://github.com/microsoft/winget-pkgs/issues/30134)